assets/scss/components/stencil/applePay/_applePay.scss

```diff
@@ -0,0 +1,54 @@
+$apple-pay-black: black;
+$apple-pay-white: white;
+
+.apple-pay-checkout-button {
```

Comment:
`applePay` should be the component name, with dashes used to separate subcomponents. So `.applePay-checkoutButton`, or `.applePay-checkout-button` if `button` is a subcomponent of `.applePay-checkout`.